src/ansys/additive/core/additive.py

```diff
@@ -612,6 +612,45 @@ def tune_material(
         MaterialTuningSummary
             Summary of material tuning.
         """  # noqa: E501
+
+        task = self.tune_material_async(input, out_dir)
+        task.wait(progress_handler=progress_handler)
+        operation = task._long_running_op
+        response = TuneMaterialResponse()
+        operation.response.Unpack(response)
+        if not response.HasField("result"):
+            raise Exception("Material tuning result not found")
+
+        return MaterialTuningSummary(input, response.result, out_dir)
```

Comment:
Just wondering why the code inside `simulation_task` was not used? It should have all this in the _unpack_summary method so that the client just calls `task.summary`. However, it was never fully tested. I also see this was just code that was moved, so maybe `simulation_task` may have not been fully implemented at the time.